src/actions/GraphFilterActions.ts

```diff
@@ -8,6 +8,7 @@ enum GraphFilterActionKeys {
   SET_FIND_VALUE = 'SET_FIND_VALUE',
   SET_GRAPH_TYPE = 'SET_GRAPH_TYPE',
   SET_HIDE_VALUE = 'SET_HIDE_VALUE',
+  SET_SHOW_UNUSED_NODES = 'SET_SHOW_UNUSED_NODES',
```

Comment:
This is OK but going forward we may just want to have one "display" query param that can handle all of the display options.  Having bookmarkability for just one option is a little weak.